src/pages/joinGameLobby/[id].js

```diff
@@ -99,24 +119,48 @@ export default function Lobby() {
 
   useEffect(() => {}, [count]);
   useEffect(() => {
+    //if they are not Host
     if (getCookie("isHost") === "false") {
       setButtonDisabled(true);
+    } else{
+      //add a POST request to update users in lobby
     }
+
+    const urlCode = window.location.pathname.split('/');
+    console.log(urlCode[urlCode.length -1])
     setGameLobbyText(urlCode[urlCode.length -1])
+    getUserName();
   }, []);
 
   let copyClick = () => {
     navigator.clipboard.writeText(gameLobbyText);
     setCopy(true);
     setCopyWord("Copied !");
   };
+useEffect(() => {
+  if(userName !== undefined){
+    setTimeout(() => {
+      setInterval(function () {getUserNames(gameLobbyText)}, 5000);
+    }, 3000)
+  } else{
```

Comment:
both sides of this "if/else" do the same thing, if you base the "if" clause on gameLobbyText existing instead of username it won't send requests to an empty endpoint